procedures/candlepin/release.md.erb

```diff
@@ -0,0 +1,10 @@
+# Build Package
+
+- [ ] Acquire Candlepin SRPM for <%= release %>
+- [ ] Build into Copr
+  - `copr build @theforeman/candlepin-staging-<%= release %> <SRPM>`
```

Comment:
Can the same SRPM be used for el8 and 9?

Reply:
Yes.